sdk-extensions/aws/src/main/java/io/opentelemetry/sdk/extension/aws/trace/AwsXrayIdGenerator.java

```diff
@@ -19,7 +19,10 @@
  * @see <a
  *     href="https://docs.aws.amazon.com/xray/latest/devguide/xray-api-sendingdata.html#xray-api-traceids">Generating
  *     Trace IDs</a>
+ * @deprecated Please use the AwsXrayIdGenerator from the
+ *     https://github.com/open-telemetry/opentelemetry-java-contrib project.
  */
+@Deprecated
```

Comment:
Can we deprecate in next release so it's after contrib c artifact of published?

Reply:
I'd be fine with that. Happy to revert that bit.

Reply:
done